test/fgpu/test_engine.py

```diff
@@ -933,6 +937,60 @@ async def test_missing_heaps(
             == np.sum(~dst_present) * n_substreams
         )
 
+        # Information needed to check that the dig-rms-dbfs sensor is correctly
+        # going into FAILURE when there are missing samples.
+        time_converter = TimeConverter(SYNC_TIME, ADC_SAMPLE_RATE)
```

Comment:
It's no longer showing up on this revision, but I previously suggested making this a TIME_CONVERTER constant at the top of the file to avoid repeating this statement multiple times across tests.